services/workflows-service/src/app.module.ts

```diff
@@ -126,6 +127,13 @@ export const validate = async (config: Record<string, unknown>) => {
     RuleEngineModule,
     NotionModule,
     SecretsManagerModule,
+    BullModule.forRoot({
+      connection: {
```

Comment:
its wrong way to init a module in NestJS also i would recommend  also to use: `config.get('redis')` 
in case we would like to include more options like ENFORCE SSL or any other configuration that the redis client support